mocks/pipe.go

```diff
@@ -48,7 +48,8 @@ func (p *Pipe) Forward(msg *streams.Message) error {
 	record := p.expectForward[0]
 	p.expectForward = p.expectForward[1:]
 
-	if msg.Key != record.key || msg.Value != record.value {
+	if (record.key != Anything && msg.Key != record.key) ||
```

Comment:
Isn't it equal to `record.key != Anything && (msg.Key != record.key || msg.Value != record.value)` which is shorter...

Reply:
No, cause you may want to just disregard to key or value separately.